### ISSUE TITLE
Allow choice of correctness on this/that quizzes

### ISSUE THREAD

**Issue by mikeyaworski** [mikeyaworski/ABS#44]
Instead of just "random" or "all correct", use a dropdown with the following options:

- Random
- 100%
- One wrong
- 90%
- 75%
- 50%

And see if the wrong selections can be predetermined in a random way.

**Comment by burritosoftware** [comment]
Is there any plan to add this eventually? Missing out on points with this or that is kinda frustrating with the random picker. Also random delay for this would be cool.

**Comment by mikeyaworski** [comment]
@burritosoftware Yes, when I get some free time. I'm quite busy with work and school at the moment. All open issues are intended to be done.

And just an FYI, you can turn off the random picker and get 100%.

**Comment by burritosoftware** [comment]
Oh neat! How do I turn off the random picker then? Do I just leave it unchecked?

**Comment by mikeyaworski** [comment]
Just uncheck "Randomly guess this/that quizzes" on the popup and it should choose the correct answers.
